gems/pending/appliance_console/service_group.rb

```diff
@@ -49,7 +49,7 @@ def stop
     private
 
     def enable_miqtop
-      LinuxAdmin.run("chkconfig", :params => {"--add" => "miqtop"})  # Is this really needed?
+      LinuxAdmin::Service.new("miqtop").enable
```

Comment:
Agreed.  I'll try to pick out any other remaining `chkconfig` / `service` calls in that one as well.